librz/core/canalysis.c

```diff
@@ -7403,3 +7403,87 @@ RZ_IPI void rz_core_analysis_value_pointers(RzCore *core, RzOutputMode mode) {
 	free(tmp);
 	rz_config_set_i(core->config, "search.align", o_align);
 }
+
+static bool __nonreturn_print_commands(void *p, const char *k, const char *v) {
+	RzCore *core = (RzCore *)p;
+	if (!strncmp(v, "func", strlen("func") + 1)) {
+		char *query = sdb_fmt("func.%s.noreturn", k);
+		if (sdb_bool_get(core->analysis->sdb_types, query, NULL)) {
+			rz_cons_printf("tnn %s\n", k);
```

Comment:
Good point, I will add it in my TODO. But it's out of the scope for the current PR, imho.

Reply:
I was also thinking about creating some kind of small API like PJ for forming Rizin commands, and the same for the C-like output.

Reply:
I think that should be discussed thoroughly  before starting with an implementation because I think it's something we should use in many places... I honestly don't like how PJ is always passed everywhere, because it makes it kinda "special" and you have to deal with it in too many places. `RzTable` might be a good step toward having a unified interface for output, but as we said in the past, many times the format of Json/table/simple output is too different... We should try to think about a very good solution.

Reply:
Created an issue for that to track and discuss. Feel free to edit/improve: https://github.com/rizinorg/rizin/issues/727